src/index.html

```diff
@@ -12,7 +12,57 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars__body">
+
+    <container class="stars__container">
```

Comment:
Use semantic tags where possible. In this case, instead of a generic `<container>`, which is not a standard HTML element, you should use a `<div>` or a more appropriate semantic element like `<section>` if it represents a thematic grouping of content.